README.md

```diff
@@ -317,6 +317,45 @@ now keep this in mind:
 The CLI text editor we all know, just in case you need to inspect some bug in
 hot deployments.
 
+### `install-addons`
+
+This script will iterate the `addons.yml` file to gather support addons, and
+subsequently install to the Odoo instance. It will then exit.
+
+This script is great to use during buildout in order to prepare an instance.
+Below is a minimalistic `docker-compose` file illustrating a use case:
+
+```yaml
+version: '2'
+
+services:
+
+  install:
+    image: laslabs/odoo-buildouts:dev-10
```

Comment:
use:

        build:
            context: ./odoo

Remember that this project is tightly coupled with its scaffolding, adding more unknown images as this to the examples adds confusion and diverges from the original purpose.

(I'm not saying your approach is wrong, but just want to keep the project consistent 😊 )

You can grab any docker-compose.yaml samples you need from the combination of [common.yaml](https://github.com/Tecnativa/docker-odoo-base/blob/scaffolding/common.yaml) and [prod.yaml](https://github.com/Tecnativa/docker-odoo-base/blob/scaffolding/prod.yaml)